From a0047b491a2b27ce1c83fe13d4f3fa943bfd3def Mon Sep 17 00:00:00 2001
From: Pradipta Ghosh <pradghos@in.ibm.com>
Date: Tue, 16 Feb 2021 18:13:33 -0800
Subject: [PATCH] BUG: fix `gaussian_kernel_estimate` on S390X

Reference issue

https://github.com/scipy/scipy/issues/13512

What does this implement/fix?

used `math.power()` over exponentiation operator to fix unexpected behavior on S390X.

Additional information

https://github.com/scipy/scipy/issues/13512#issuecomment-780246692
---
 scipy/stats/_stats.pyx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scipy/stats/_stats.pyx b/scipy/stats/_stats.pyx
index d331c1a201b..1b8d4dba8bb 100644
--- a/scipy/stats/_stats.pyx
+++ b/scipy/stats/_stats.pyx
@@ -566,7 +566,7 @@ def gaussian_kernel_estimate(points, values, xi, precision, dtype, real _=0):
     values_ = values.astype(dtype, copy=False)
 
     # Evaluate the normalisation
-    norm = (2 * PI) ** (- d / 2)
+    norm = math.pow((2 * PI) ,(- d / 2))
     for i in range(d):
         norm *= whitening[i, i]
 
